### ISSUE TITLE
vSphere Remote Control feature

### ISSUE THREAD

**Issue by Martyx00** [cuckoosandbox/cuckoo#2444]
Thanks for creating an issue! But first: did you read our community guidelines?
https://cuckoo.sh/docs/introduction/community.html

##### My issue is:
Feature request: Remote Control support for vSphere machinery

Do you plan to implement support for Remote Control for vSphere machinery in future releases?

Thanks

##### My Cuckoo version and operating system are:
2.0.6
##### This can be reproduced by:
N/A
##### The log, error, files etc can be found at:
```
2018-08-22 03:51:50,456 [cuckoo.core.scheduler] ERROR: Failure in AnalysisManager.run
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/scheduler.py", line 737, in run
    self.launch_analysis()
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/scheduler.py", line 474, in launch_analysis
    "Remote control support has not been implemented "
CuckooMachineError: Remote control support has not been implemented for this machinery.
```

**Comment by RicoVZ** [comment]
Hi Martyx00,

Thanks for posting an issue.

Remote control for vSphere is not currently on our to-do list or roadmap. Obviously we want to support it for as many machinery modules as possible. I will ask this to be marked as a feature request.

By the way: the behavior you are seeing now (the analysis aborting when remote control is not supported), will be fixed in 2.1.

**Comment by RicoVZ** [comment]
Thanks for sharing @Martyx00. I am sure it will help in supporthing this in the future.
I also just heard that we are currently developing a way to also staticly configure a rdp or vnc url for a VM. This will also allow remote control usage on most unsupported platforms.

This will likely also be released in 2.1.

**Comment by Martyx00** [comment]
I am closing the issue and looking forward for the release 2.1 :)

Thanks
